docs/examples/getting-started/quickstart.py

```diff
@@ -43,8 +43,8 @@
 table = postgres.table("product", cursor="last_modified")
 
 
-@source(table, disorder="1d", cdc="upsert", every="1m", tier="prod")
-@source(webhook.endpoint("Product"), disorder="1d", cdc="upsert", tier="dev")
+@source(table, disorder="1d", cdc="upsert", every="1m", env="prod")
+@source(webhook.endpoint("Product"), disorder="1d", cdc="upsert", env="dev")
```

Comment:
Could you plz remove `env` from quickstart example and just leave one decorator each? Otherwise makes the quickstart look more daunting/confusing.